pitest-maven-verification/src/test/java/org/pitest/PitMojoIT.java

```diff
@@ -149,13 +157,15 @@ public void shouldCorrectlyTargetTestsWhenMultipleBlocksIncludeALine() throws Ex
    */
   @Test
   public void shouldSkipSiteReportGeneration() throws Exception {
-    File testDir = prepare("/pit-site-skip");
-
-    verifier.executeGoals(Arrays.asList("test", "org.pitest:pitest-maven:mutationCoverage", "site"));
-
+    File testDir = prepareSiteTest("/pit-site-skip");
```

Comment:
I started by putting in comments for each test scenario I wanted to cover and then added the tests under each comment.  I left the comments in place as another form of documentation, but I think at this point I could delete all the comments.  Do you see any value in keeping them?  If not, I will remove them.  I did rename a few tests to better clarify what they are testing.